source/codegen/metadata/nidaqmx/functions.py

```diff
@@ -8,23 +7,142 @@
+                'name': 'task',
+                'type': 'TaskHandle'
+            },
+            {
+                'direction': 'in',
+                'name': 'physicalChannel',
+                'type': 'const char[]'
+            },
+            {
+                'direction': 'in',
+                'name': 'nameToAssignToChannel',
+                'type': 'const char[]'
+            },
+            {
+                'direction': 'in',
+                'enum': 'InputTermCfgWithDefault',
+                'name': 'terminalConfig',
+                'type': 'int'
+            },
+            {
+                'direction': 'in',
+                'name': 'minVal',
+                'type': 'float64'
+            },
+            {
+                'direction': 'in',
+                'name': 'maxVal',
+                'type': 'float64'
+            },
+            {
+                'direction': 'in',
+                'enum': 'VoltageUnits2',
+                'name': 'units',
+                'type': 'int'
+            },
+            {
+                'direction': 'in',
+                'name': 'customScaleName',
+                'type': 'const char[]'
+            }
+        ],
+        'returns': 'int32'
+    },
+    'CreateDIChan': {
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'task',
+                'type': 'TaskHandle'
+            },
+            {
+                'direction': 'in',
+                'name': 'lines',
+                'type': 'const char[]'
+            },
+            {
+                'direction': 'in',
+                'name': 'nameToAssignToLines',
+                'type': 'const char[]'
+            },
+            {
+                'direction': 'in',
+                'enum': 'LineGrouping',
+                'name': 'lineGrouping',
+                'type': 'int'
+            }
+        ],
+        'returns': 'int32'
+    },
+    'CreateDOChan': {
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'task',
+                'type': 'TaskHandle'
+            },
+            {
+                'direction': 'in',
+                'name': 'lines',
+                'type': 'const char[]'
+            },
+            {
+                'direction': 'in',
+                'name': 'nameToAssignToLines',
+                'type': 'const char[]'
+            },
+            {
+                'direction': 'in',
+                'enum': 'LineGrouping',
+                'name': 'lineGrouping',
+                'type': 'int'
+            }
+        ],
+        'returns': 'int32'
     },
     'CreateTask': {
         'init_method': True,
         'parameters': [
             {
                 'direction': 'in',
+                'is_session_name': True,
                 'name': 'sessionName',
-                'type': 'const char*',
-                'is_session_name': True
+                'type': 'const char[]'
```

Comment:
I changed these to `char[]` because it matches what the daq header uses and because I didn't see `*` used in any types in the rest of the grpc-metadata (though there are some typedefs of pointer types) and they're typically associated with outputs in the codegen.